<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCar</class>
 <widget class="QDialog" name="DialogCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nome do Arquivo:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LECaminho"/>
     </item>
     <item>
      <widget class="QPushButton" name="BSeleciona">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CBSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Entrada(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListView" name="LVEntradas"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Saida(s)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListView" name="LVSaida"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Instante para iniciar a coleta dos dados&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;OBS.: &amp;quot;0&amp;quot; =  coleta do inicio do arquivo &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>T. Inicial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LETIni">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Instante para iniciar a coleta dos dados&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;OBS.: &amp;quot;0&amp;quot; =  coleta do inicio do arquivo &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBTIni">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>seg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>us</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Instante para finalizar a coleta dos dados&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;OBS.: &amp;quot;0&amp;quot; =  coleta até o fim do arquivo &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>T.  Final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LETFim">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Instante para finalizar a coleta dos dados&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;OBS.: &amp;quot;0&amp;quot; =  coleta até o fim do arquivo &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="maxLength">
        <number>9999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CBTFim">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>seg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>us</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="BConcatenar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Concatenar</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BCarregar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Carregar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="DIC_pb">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="LVStBar">
       <property name="text">
        <string>Aguardando Usuário...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="LVStBar2">
       <property name="text">
        <string>Linhas: 0, Colunas: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BCarregar</sender>
   <signal>clicked()</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_Carregar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BSeleciona</sender>
   <signal>clicked()</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_Caminho()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LECaminho</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_Indicar(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BConcatenar</sender>
   <signal>clicked()</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_Concatenar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LVSaida</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_ModelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogCar</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogCar</sender>
   <signal>finished(int)</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_Finalizar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LVEntradas</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DialogCar</receiver>
   <slot>slot_UL_ModelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_UL_Carregar()</slot>
  <slot>slot_UL_Caminho()</slot>
  <slot>slot_UL_Indicar(QString)</slot>
  <slot>slot_UL_Finalizar()</slot>
  <slot>slot_UL_Concatenar()</slot>
  <slot>slot_UL_SalvarArquivo()</slot>
  <slot>slot_UL_ModelClicked()</slot>
 </slots>
</ui>
